CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
> used extensively from Slicer

Look like there are used in few modules and in one file from the core

```
$ ack --cpp -l vnl
Modules/CLI/LabelMapSmoothing/LabelMapSmoothing.cxx
Modules/CLI/RobustStatisticsSegmenter/SFLSRobustStatSegmentor3DLabelMap_single.txx
Modules/CLI/RobustStatisticsSegmenter/SFLSSegmentor3D.txx
Modules/CLI/RobustStatisticsSegmenter/SFLS.h
Modules/CLI/ExpertAutomatedRegistration/itkAnisotropicSimilarity3DTransformTest.cxx
Modules/CLI/ExpertAutomatedRegistration/ITKRegistrationHelper/itkAnisotropicSimilarityLandmarkBasedTransformInitializer.txx
Modules/CLI/ExpertAutomatedRegistration/ITKRegistrationHelper/itkAnisotropicSimilarity3DTransform.txx
Modules/CLI/ExpertAutomatedRegistration/ITKRegistrationHelper/itkImageRegionMomentsCalculator.txx
Modules/CLI/ExpertAutomatedRegistration/ITKRegistrationHelper/itkImageRegionMomentsCalculator.h
Modules/CLI/ExpertAutomatedRegistration/ITKRegistrationHelper/itkRigidImageToImageRegistrationMethod.txx
Modules/CLI/PETStandardUptakeValueComputation/itkDCMTKFileReader.h
Modules/CLI/PETStandardUptakeValueComputation/itkDCMTKFileReader.cxx
Modules/CLI/ResampleDTIVolume/dtiprocessFiles/dtitypes.h
Modules/CLI/ResampleDTIVolume/itkWarpTransform3D.h
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DAbsCorrection.h
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DFSAffineTransform.h
Modules/CLI/ResampleDTIVolume/itkMatrixExtended.h
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DNearestCorrection.h
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DFSAffineTransform.txx
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DPPDAffineTransform.txx
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DPPDAffineTransform.h
Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DZeroCorrection.h
Libs/MRML/Core/vtkMRMLSliceNode.cxx
```

> ITK dependencies

Ultimately these dependencies internal to ITK wouldn't "leak" into public header.

Cc: @thewtex